src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
 	  <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-	  <VersionPrefix>12.1.0</VersionPrefix>

```

Comment:
Since this is only adding new methods it does not require a major version bump, should be version 12.2.0

Reply:
yep, only breaking changes need a major version bump per https://semver.org/